README.md

```diff
@@ -0,0 +1,180 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 Catalog of my things](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🎥 Vedio Description](#vd-disc)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of my things <a name="about-project"></a>
+
+This project is console app built using ruby, the app will help you to keep a record of different types of things you own: books, music albums, movies, and games, the data stored in json file, and it will be always useable whenever you start the app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+- The app built using [Ruby](https://www.ruby-lang.org/en/).
+- The **schema** file built using **SQL**.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The app help you to keep recods about any items you own, thing like
+
+- **Store and Display Games**
+- **Store and Display Music Albums**
+- **Store and Display Books**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Video Description <a name="vd-disc"></a>
+
+In this recorded video, we describe how we built this amazing project as a team.
+
+[Project presentation video](https://drive.google.com/file/d/1G5wzKzUCcXMvgMBkqsKY5xbX5VSqiBtm/view?usp=sharing)
```

Comment:
- [ ] [REQUIRED] Hello Team, great presentation I must say 🥇 , however, it is a hard requirement that the length of the presentation video must not exceed five (5) minutes. The current presentation has a length of 9 minutes and 4 seconds. Let's ensure that we stick to the project requirement. Also, the video must contain a highlight about a piece of code each student is proud of making.